docker/cp-test

```diff
@@ -27,87 +27,17 @@ export AUTOCONF=1
 export FORCECERT=1
 export UNITTEST=0
 
-# functions to abstract away buildr vs. gradle differences
-build_clean() {
```

Comment:
We can't remove this stuff from cp-test because its packaged in the container images, which are used by older branches

Reply:
You can at least backport this change to candlepin-4.2-HOTFIX (which was cut just yesterday from master), and maybe even candlepin-4.1-HOTFIX, which will make us get rid of this stuff earlier in the future (in about ~2 months when 4.0 goes out of support).